app/cdap/components/RecipeList/RecipeTableRow.tsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2022 Cask Data, Inc.
+ * Copyright © 2023 Cask Data, Inc.
```

Comment:
this does not need to be changed to 2023
this file is already merged and we are just doing some updates on this
only new files will have 2023
plz update across PR

Reply:
changed to 2022